packages/sdk/src/contracts/prebuilt-implementations/edition-drop.ts

```diff
@@ -225,6 +228,19 @@ export class EditionDrop extends StandardErc1155<DropERC1155> {
     return this.erc1155.totalCount();
   }
 
+  /**
+   * Get current owner of the contract
+   *
+   * @example
+   * ```javascript
+   * const owner = await contract.getOwner();
+   * ```
+   * @returns The owner address.
+   */
+  public async getOwner(): Promise<string> {
```

Comment:
you don't need these top level functions. Let's just expose it via the namespace: `contract.owner.get()` directly